modules/acquia_cms_dam/composer.json

```diff
@@ -33,6 +33,13 @@
     "extra": {
         "branch-alias": {
             "dev-develop": "1.x-dev"
+        },
+        "enable-patching": true,
+        "patches": {
+            "drupal/acquia_dam": {
```

Comment:
@rajeshreeputra Let's not add this patch in here. Rather add this patches in root composer.json. As Acquia maintain this module, we can connect with DIT team to prioritise release of this.

Reply:
sure